enhancements/etcd/assisted-install.md

```diff
@@ -0,0 +1,221 @@
+---
+title: support-assisted-installer
+
+authors:
+  - "@hexfusion"
+reviewers:
+  - "@deads2k"
+  - "@skolicha"
+  - "@dmace"
+approvers:
+  - TBD
+creation-date: 2020-09-16
+last-updated: 2020-09-16
+status: provisional
+---
+
+# backup config controller
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+The assisted installer was designed to utilize three bare-metal machines to
+provision the control-plane. In order to do this after bootstrap is complete
+the bootstrap node is repurposed into the third master node.
```

Comment:
Maybe it should be explicitly mentioned that the assisted installer requires the support of etcd with 2 hosts for only a short period, until the bootstrap host has rebooted and joined the cluster. after installation is complete, there will be 3 hosts in the etcd cluster